_rules/aria-required-context-role-ff89c9.md

```diff
@@ -109,19 +109,6 @@ These elements with an [explicit role][] of `listitem` are children in the [acce
 
 #### Passed Example 5
 
-These elements with an [explicit role][] of `listitem` are children in the [accessibility tree][] of an element with their [required context role][] even though they are not its DOM children. The `aria-owns` attribute is used to alter the accessibility tree and place the target elements in their [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
-
-```html
-<div role="list" aria-owns="item1 item2">
-	<div role="navigation">
-		<div id="item1" role="listitem">List item 1</div>
-		<div id="item2" role="listitem">List item 2</div>
-	</div>
-</div>
-```
```

Comment:
I think this example is important to show how `aria-owns` twists the accessibility tree and that the rule cares about this (rather than the DOM tree). OTOH, I do agree that the resulting accessibility tree is "bad" due to the `<div role="navigation">` being a child of the list. maybe replace the example with:
```html
<div role="list" aria-owns="item1 item2">
</div>
<div role="navigation">
	<div id="item1" role="listitem">List item 1</div>
	<div id="item2" role="listitem">List item 2</div>
</div>
```
Which is arguably more artificial, but would pass the other rule 🤔

Reply:
I see your point. I think that the alternative you gave would be better since it's a passed example.